Student/Student/Quizzes/QuizList/QuizListPresenter.swift

```diff
@@ -48,14 +44,49 @@ class QuizListPresenter {
         self.view = view
     }
 
+    func sectionOrder(_ name: String) -> Int {
+        switch QuizType(rawValue: name) {
+        case .assignment?: return 0
+        case .practice_quiz?: return 1
+        case .graded_survey?: return 2
+        case .survey?: return 3
+        case .none: return 4
+        }
+    }
```

Comment:
I realized this morning that I could prefix this onto the `order` attribute, then I wouldn't need the `sectionIndex` section remapping, because the sections would be in the right order already. Does that seem better than having this logic here?

Reply:
If you prefixed it on order, some of this view logic would have to move to the ManagedObject right?  I think i like the logic living here if i had to choose.  I'd say leave it.